### DISCUSSION TITLE
Welcome to OpenSeadragon Discussions!

### DISCUSSION THREAD

**Discussion by iangilman** [openseadragon/openseadragon#1920] [General]
Looks like GitHub has a new discussion feature… Let's give it a try! :-)

**Comment by xgui3783** [comment]
So, built in glitter? 😯

**Reply by iangilman** [reply]
Something like that… :-)

I figure a lot of our issues end up being more just questions, so maybe having them over here in discussions would be good.

Also, this might be a good place for announcements. We shall see!

**Comment by ruqayya** [comment]
Hi,

I posted a question a few days ago. Here is the link https://github.com/openseadragon/openseadragon/issues/1946#issue-819735836.

I am not able to figure out how to access the tiles currently displayed.

**Reply by iangilman** [reply]
@ruqayya sorry for the delayed response! I've been slowly trying to catch up here...
